web_src/js/components/DashboardRepoList.vue

```diff
@@ -77,8 +77,10 @@
                 <svg-icon name="octicon-archive" :size="16"/>
               </div>
             </a>
-            <!-- the commit status icon logic is taken from templates/repo/commit_status.tmpl -->
-            <svg-icon v-if="repo.latest_commit_status_state" :name="statusIcon(repo.latest_commit_status_state)" :class-name="'gt-ml-3 commit-status icon text ' + statusColor(repo.latest_commit_status_state)" :size="16"/>
+            <a class="repo-list-status-link" v-if="repo.latest_commit_status_state" :href="repo.latest_commit_status_state_link" :data-tooltip-content="repo.latest_commit_status_state">
```

Comment:
No idea why it needs this style.

Its parent is "flex + align-items: center", if it needs padding, the padding can be added on the parent container.